packages/calcite-components/src/components/card/card.tsx

```diff
@@ -290,39 +332,48 @@ export class Card
   }
 
   private renderHeader(): VNode {
-    const { el } = this;
-    const heading = getSlotted(el, SLOTS.heading);
-    const description = getSlotted(el, SLOTS.description);
-    const hasHeader = heading || description;
-    const subtitle = getSlotted(el, SLOTS.subtitle);
-    const title = getSlotted(el, SLOTS.title);
-    const hasDeprecatedHeader = subtitle || title;
-    return hasHeader || hasDeprecatedHeader ? (
-      <header class={CSS.header}>
+    const hasHeader = this.hasHeading || this.hasDescription;
+    const hasDeprecatedHeader = this.hasSubtitle || this.hasTitle;
+    const showHeader = hasHeader || hasDeprecatedHeader;
+
+    return (
+      <header class={CSS.header} hidden={!showHeader}>
         {this.selectable ? this.renderCheckboxDeprecated() : null}
         <div class={CSS.headerTextContainer}>
-          <slot key="heading-slot" name={SLOTS.heading} />
-          <slot key="description-slot" name={SLOTS.description} />
-          <slot key="deprecated-title-slot" name={SLOTS.title} />
-          <slot key="deprecated-subtitle-slot" name={SLOTS.subtitle} />
+          <slot
+            key="heading-slot"
```

Comment:
`key` isn't needed since this element and its siblings are rendered statically (see https://stenciljs.com/docs/templating-jsx#conditionals and https://stenciljs.com/docs/templating-jsx#dealing-with-children).